ci/pipeline.yml

```diff
@@ -21,14 +21,14 @@ groups:
   - cf-api-controllers-unit-tests
   - registry-buddy-unit-tests
   - registry-buddy-integration-tests
-  - backup-metadata-unit-tests
-  - build-backup-metadata-image
+  - backup-metadata-generator-generator-unit-tests
+  - build-backup-metadata-generator-generator-image
```

Comment:
I'm guessing this is a typo and should just be `generator-`, not `generator-generator-`

Reply:
yes, thank you for catching this 🙇‍♂️

Reply:
fixed and amended the commit
https://github.com/cloudfoundry/capi-ci/pull/16/commits/5ea7abc3e0c99eda0b702ee3db1e96de39a80c3f